content/docs/dvclive/api-reference/next_step.md

```diff
@@ -0,0 +1,43 @@
+# dvclive.next_step()
+
+Signals that the current step has ended and saves _summary metrics_ to a
+`.json`.
```

Comment:
```suggestion
Signals that the current step has ended and optionally saves _summary metrics_ to a
`.json`.
```

This also can do more, right? Like trigger the html report or make a checkpoint. Specifying everything `next_step` can do in the description might be challenging to maintain. Is there a way to summarize, or maybe refer back to `init` for details? What do you think?

Reply:
Good point. Probably refer back to `init` would be nice

Reply:
Updated with a reference to `init parameters`. Not sure about the current state though as I still specify "everything" in the `Description` section bellow.

Reply:
Would it make sense to move the different behaviors (i.e. enabling/disabling summary/html/checkpoints) to "Example" sections, like in https://dvc.org/doc/api-reference/open#example-use-data-or-models-from-a-dvc-repository ? 

@jorgeorpinel

Reply:
That's a good idea. I'm not sure if those belong here or in `init`, but as long as we link between them, either is probably fine.